src/frontends/tensorflow/tests/test_models/models_pbtxt/stateful_partitioned_call.pbtxt

```diff
@@ -0,0 +1,240 @@
+node {
```

Comment:
This model is called `stateful_partitioned_call`, but I don't see a `StatefulPartitionedCall` operation in the graph.

Reply:
There is `PartitionedCall` operation in the graph. I will rename it. I didn't manage to generate a graph with `StatefulPartitionedCall` operation.

Reply:
Renamed.